src/main.tsx

```diff
@@ -40,13 +42,15 @@ ReactDOM.createRoot(root).render(
             wind={[-0.5, 0.5]}
             radius={[14.0, 16.0]}
             style={{
-              zIndex: 1,
+              zIndex: 12,
```

Comment:
Why is this z-index changed?

Reply:
The cherry blossom effect was initially positioned below the Circle component on the RouteMap due to its z-index, so I've adjusted this. With the upcoming update, the z-index order will be set as follows: the Line component at z-index 0, Circle at z-index 1, and the cherry blossom effect (snowfall) at z-index 2.

Reply:
If that's the case, the zIndex value should be 2, not 12.
```suggestion
              zIndex: 2,
```